extension/jdtls.ext/com.microsoft.gradle.bs.importer/src/com/microsoft/gradle/bs/importer/GradleBuildServerBuildSupport.java

```diff
@@ -78,6 +79,36 @@ public boolean applies(IProject project) {
         return Utils.isGradleBuildServerProject(project);
     }
 
+    @Override
+    public void update(IProject project, boolean force, IProgressMonitor monitor) throws CoreException {
+        if (!applies(project)) {
+            return;
+        }
+
+        boolean shouldUpdate = force || GradleBuildServerProjectImporter.updateConfigurationDigest(project);
```

Comment:
Should be `updateConfigurationDigest(...) || force`. Should update digest on every update.

Reply:
Ah, yes!